src/components/UserSettings/GlobalSettings.js

```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux'
 import { push } from 'connected-react-router'
 import { saveGlobalOptions } from '_/actions'
 import { FormControl, Switch } from 'patternfly-react'
-import { withMsg, localeWithFullName, DEFAULT_LOCALE } from '_/intl'
+import { withMsg, localeWithFullName, locale, DEFAULT_LOCALE } from '_/intl'
```

Comment:
`locale` should not be accessed directly (similar to `msg` object) - both are provided by `withMsg` HOC (check other usages in the code)